bundle/edu.gemini.phase2.skeleton.servlet/src/main/scala/edu/gemini/phase2/skeleton/factory/SpProgramFactory.scala

```diff
@@ -70,13 +70,13 @@ object SpProgramFactory {
     prog.setThesis(isThesis(proposal))
 
     prog.setPIInfo(piInfo(proposal))
-    hostNgoEmail(proposal) foreach { e => prog.setPrimaryContactEmail(e) }
+    hostNgoEmail(proposal) foreach prog.setPrimaryContactEmail
 
     // Note: not a typo -- "contact person" is an email
     gemEmail(proposal) foreach { e => prog.setContactPerson(e) }
 
-    minBand3Time(proposal) foreach { tv => prog.setMinimumTime(tv) }
-    timeAcctAllocation(proposal) foreach { alloc => prog.setTimeAcctAllocation(alloc) }
+    minBand3Time(proposal) foreach prog.setMinimumTime
+    timeAcctAllocation(proposal) foreach prog.setTimeAcctAllocation
```

Comment:
I think for now we should just scale the minimum time and store it, since before we were just storing it without any validation?   In theory this validation should be done upstream but if we want to check the proposal again before ingesting it we might want to do so with a bit more time to think it all through.